src/libprojectM/ProjectM.hpp

```diff
@@ -110,7 +110,7 @@ class PROJECTM_EXPORT ProjectM
      */
     void LoadPresetData(std::istream& presetData, bool smoothTransition);
 
-    void ResetOpenGL(size_t width, size_t height);
+    void ResetOpenGL(int width, int height);
```

Comment:
Could just change this (and subsequent members etc. storing the window size) to `uint32_t`, as negative window sizes don't make much sense. The API also uses `size_t`, so there's no reason to make it signed.

Reply:
Good catch. The members are already uint32_t, I must have just missed the prototype here.